<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>525</width>
    <height>235</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>525</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Go To Setpoint</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resources/nSOTScannerResources.qrc">
    <normaloff>:/nSOTScanner/Pictures/jj_schema.svg</normaloff>:/nSOTScanner/Pictures/jj_schema.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget{background-color: rgb(2, 2, 2);}
#line_11{color: rgb(168,168,168)}
#line_12{color: rgb(168,168,168)}
#line_13{color: rgb(168,168,168)}
#line_14{color: rgb(168,168,168)}
#line_15{color: rgb(168,168,168)}
#line_16{color: rgb(168,168,168)}
#line_17{color: rgb(168,168,168)}
#line_18{color: rgb(168,168,168)}
#line_19{color: rgb(168,168,168)}
#line_20{color: rgb(168,168,168)}
#line_21{color: rgb(168,168,168)}
#line_22{color: rgb(168,168,168)}
#line_23{color: rgb(168,168,168)}
#line_24{color: rgb(168,168,168)}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="biasPntsLine">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>145</y>
      <width>45</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>500</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="Line" name="line_13">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>95</y>
      <width>3</width>
      <height>174</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#line_13{color: rgb(168,168,168)}
#line_11{color: rgb(168,168,168)}
#line_12{color: rgb(168,168,168)}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>110</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#a8a8a8;&quot;&gt;(V)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zeroBiasBtn">
    <property name="geometry">
     <rect>
      <x>102</x>
      <y>246</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed#zeroBiasBtn{
color: rgb(95,107,166);
background-color:rgb(168,168,168);
border: 1px solid rgb(95,107,166);
border-radius: 5px
}

QPushButton#zeroBiasBtn{
color: rgb(95,107,166);
background-color:rgb(0,0,0);
border: 2px solid  rgb(95,107,166);
border-radius: 5px
}</string>
    </property>
    <property name="text">
     <string>Zero
Bias</string>
    </property>
   </widget>
   <widget class="Line" name="line_16">
    <property name="geometry">
     <rect>
      <x>222</x>
      <y>266</y>
      <width>37</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#line_9{color: rgb(168,168,168);}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>110</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#a8a8a8;&quot;&gt;Bias Setpoint:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="blinkBtn">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>40</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed#blinkBtn{
color: rgb(95,107,166);
background-color:rgb(168,168,168);
border: 1px solid rgb(95,107,166);
border-radius: 5px
}

QPushButton#blinkBtn{
color: rgb(95,107,166);
background-color:rgb(0,0,0);
border: 2px solid  rgb(95,107,166);
border-radius: 5px
}</string>
    </property>
    <property name="text">
     <string>Blink</string>
    </property>
   </widget>
   <widget class="QLabel" name="currBiasLbl">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>215</y>
      <width>296</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#a8a8a8;&quot;&gt;Current Bias:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_readBias">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>246</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed#push_readBias{
color: rgb(95,107,166);
background-color:rgb(168,168,168);
border: 1px solid rgb(95,107,166);
border-radius: 5px
}

QPushButton#push_readBias{
color: rgb(95,107,166);
background-color:rgb(0,0,0);
border: 2px solid  rgb(95,107,166);
border-radius: 5px
}</string>
    </property>
    <property name="text">
     <string>Read
Bias</string>
    </property>
   </widget>
   <widget class="Line" name="line_12">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>95</y>
      <width>250</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#line_9{color: rgb(168,168,168);}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_17">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>266</y>
      <width>25</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#line_9{color: rgb(168,168,168);}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_38">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>180</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#a8a8a8;&quot;&gt;Step Delay:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="BiasInc">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>145</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#a8a8a8;&quot;&gt;Number of Steps:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>45</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#a8a8a8;&quot;&gt;Electronic Feedback Control&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="gotoBiasBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>246</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed#gotoBiasBtn{
color: rgb(95,107,166);
background-color:rgb(168,168,168);
border: 1px solid rgb(95,107,166);
border-radius: 5px
}

QPushButton#gotoBiasBtn{
color: rgb(95,107,166);
background-color:rgb(0,0,0);
border: 2px solid rgb(95,107,166);
border-radius: 5px
}</string>
    </property>
    <property name="text">
     <string>Go To 
Bias</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="biasSetpntLine">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>110</y>
      <width>45</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.000</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="biasDelayLine">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>180</y>
      <width>45</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>180</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#a8a8a8;&quot;&gt;(ms)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="Line" name="line_11">
    <property name="geometry">
     <rect>
      <x>258</x>
      <y>95</y>
      <width>3</width>
      <height>174</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#line_8{color: rgb(168,168,168)}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="push_Servers">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#push_Servers{
background: rgb(144, 140, 9);
 border-radius: 4px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="push_FdbkOn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>61</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed#push_FdbkOn{
color: rgb(95,107,166);
background-color:rgb(168,168,168);
border: 1px solid rgb(95,107,166);
border-radius: 5px
}

QPushButton#push_FdbkOn{
color: rgb(95,107,166);
background-color:rgb(0,0,0);
border: 2px solid  rgb(95,107,166);
border-radius: 5px
}</string>
    </property>
    <property name="text">
     <string>Feedback
on</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_FdbkOff">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>61</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed#push_FdbkOff{
color: rgb(95,107,166);
background-color:rgb(168,168,168);
border: 1px solid rgb(95,107,166);
border-radius: 5px
}

QPushButton#push_FdbkOff{
color: rgb(95,107,166);
background-color:rgb(0,0,0);
border: 2px solid  rgb(95,107,166);
border-radius: 5px
}</string>
    </property>
    <property name="text">
     <string>Feedback
off</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gotoGateBtn">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>246</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed#gotoGateBtn{
color: rgb(95,107,166);
background-color:rgb(168,168,168);
border: 1px solid rgb(95,107,166);
border-radius: 5px
}

QPushButton#gotoGateBtn{
color: rgb(95,107,166);
background-color:rgb(0,0,0);
border: 2px solid rgb(95,107,166);
border-radius: 5px
}</string>
    </property>
    <property name="text">
     <string>Go To 
Gate</string>
    </property>
   </widget>
   <widget class="Line" name="line_14">
    <property name="geometry">
     <rect>
      <x>515</x>
      <y>95</y>
      <width>3</width>
      <height>174</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#line_8{color: rgb(168,168,168)}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="gatePntsLine">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>145</y>
      <width>45</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>500</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="Line" name="line_18">
    <property name="geometry">
     <rect>
      <x>265</x>
      <y>266</y>
      <width>35</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#line_9{color: rgb(168,168,168);}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_19">
    <property name="geometry">
     <rect>
      <x>487</x>
      <y>266</y>
      <width>30</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#line_9{color: rgb(168,168,168);}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="push_readGate">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>246</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed#push_readGate{
color: rgb(95,107,166);
background-color:rgb(168,168,168);
border: 1px solid rgb(95,107,166);
border-radius: 5px
}

QPushButton#push_readGate{
color: rgb(95,107,166);
background-color:rgb(0,0,0);
border: 2px solid  rgb(95,107,166);
border-radius: 5px
}</string>
    </property>
    <property name="text">
     <string>Read
Gate</string>
    </property>
   </widget>
   <widget class="Line" name="line_15">
    <property name="geometry">
     <rect>
      <x>265</x>
      <y>95</y>
      <width>250</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#line_9{color: rgb(168,168,168);}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>465</x>
      <y>180</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#a8a8a8;&quot;&gt;(ms)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_39">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#a8a8a8;&quot;&gt;Step Delay:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="gateSetpntLine">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>110</y>
      <width>45</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.000</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="BiasInc_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>145</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#a8a8a8;&quot;&gt;Number of Steps:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>110</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#a8a8a8;&quot;&gt;(V)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#a8a8a8;&quot;&gt;Tip Gate Setpoint:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zeroGateBtn">
    <property name="geometry">
     <rect>
      <x>368</x>
      <y>246</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed#zeroGateBtn{
color: rgb(95,107,166);
background-color:rgb(168,168,168);
border: 1px solid rgb(95,107,166);
border-radius: 5px
}

QPushButton#zeroGateBtn{
color: rgb(95,107,166);
background-color:rgb(0,0,0);
border: 2px solid  rgb(95,107,166);
border-radius: 5px
}</string>
    </property>
    <property name="text">
     <string>Zero
Gate</string>
    </property>
   </widget>
   <widget class="QLabel" name="currGateLbl">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>215</y>
      <width>296</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#a8a8a8;&quot;&gt;Current Gate:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="Line" name="line_20">
    <property name="geometry">
     <rect>
      <x>264</x>
      <y>95</y>
      <width>3</width>
      <height>174</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#line_13{color: rgb(168,168,168)}
#line_11{color: rgb(168,168,168)}
#line_12{color: rgb(168,168,168)}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="gateDelayLine">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>180</y>
      <width>45</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="Line" name="line_21">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>90</y>
      <width>511</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#line_9{color: rgb(168,168,168);}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_22">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>30</y>
      <width>511</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#line_9{color: rgb(168,168,168);}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_23">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>30</y>
      <width>3</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#line_13{color: rgb(168,168,168)}
#line_11{color: rgb(168,168,168)}
#line_12{color: rgb(168,168,168)}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_24">
    <property name="geometry">
     <rect>
      <x>515</x>
      <y>30</y>
      <width>3</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#line_13{color: rgb(168,168,168)}
#line_11{color: rgb(168,168,168)}
#line_12{color: rgb(168,168,168)}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../Resources/nSOTScannerResources.qrc"/>
 </resources>
 <connections/>
</ui>
